lua/lvim/utils/table.lua

```diff
@@ -21,4 +21,18 @@ function Table.contains(t, predicate)
   return Table.find_first(t, predicate) ~= nil
 end
 
+-- Remove item from table by it's value
+-- @param t The table
+-- @param value The item to remove
+-- @return True if item was removed, false otherwise
+function Table.remove(t, value)
```

Comment:
Mutation a function input is a bad idea. Built-in table filter has a better API in my opinion. Creating a new table and returning that is the best and safest way to proceed.